src/test/javascript/OpenLayers/Layer/NcWMSSpec.js

```diff
@@ -99,19 +99,11 @@ describe("OpenLayers.Layer.NcWMS", function() {
 
     describe("loadTimeSeriesForDay", function() {
         it("when already loaded", function() {
-            var functionCalled = false;
             spyOn(cachedLayer, '_timeSeriesLoadedForDate').andCallFake(function() { functionCalled = true });
             cachedLayer.temporalExtent.parse(['2001-07-02T00:00:00']);
 
             cachedLayer.loadTimeSeriesForDay(moment.utc('2001-07-02T00:00:00Z'));
-
-            waitsFor(function() {
-                return functionCalled;
-            }, "_timeSeriesDatesLoaded not called", 1000);
-
-            runs(function() {
-                expect(cachedLayer._timeSeriesLoadedForDate).toHaveBeenCalled();
-            });
+            expect(cachedLayer._timeSeriesLoadedForDate).toHaveBeenCalled();
```

Comment:
I'm not trying to be a pain @pmbohm but is this bit also unrelated?

I'm not trying to be difficult, it is really disconcerting seeing unrelated changes in a PR and makes it dangerous to press the merge button as you're not confident about what changes will go in. **Let me know if you want a hand sorting this out.**